src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/closure.php

```diff
@@ -54,4 +56,20 @@ protected static function getClosureService($container)
     {
         return $container->services['closure'] = (new \stdClass())->__invoke(...);
     }
+
+    /**
+     * Gets the public 'closure_of_service_closure' shared service.
+     *
+     * @return \Closure
+     */
+    protected static function getClosureOfServiceClosureService($container)
+    {
+        $containerRef = $container->ref;
+
+        return $container->services['closure_of_service_closure'] = ${($_ = #[\Closure(name: 'bar2', class: 'stdClass')] static function () use ($containerRef) {
```

Comment:
we should be able to remove the `$_` wrapper by removing the call to dumpLiteralClass, isn't it?